### PR TITLE
Revert "feat(general): Add thread.errored attribute"

### PR DESCRIPTION
**Pull request by jan-auer**
Reverts getsentry/semaphore#306

### CODE REVIEW

**Comment by jan-auer** [comment]
@bruno-garcia @marandaneto Could you please provide me with an updated android payload so I can check it in?

**Review by bruno-garcia — APPROVED** [review]
Assuming the new payload doesn't block this PR.
Can be taken [from here](https://github.com/getsentry/sentry/pull/16086) once it's merged.

**Pull request merged**
